.editorconfig

```diff
@@ -0,0 +1,13 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
EditorConfig works across IDEs / Text Editors to maintain some basic code styles.  I believe the above matches the style requirements articulated in CodeMirror's contribution style guide.